### PR TITLE
Added readme

### PR DESCRIPTION
**Pull request by buyan-m**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## [Okul.one](https://okul.one)
+Free educational platform for teachers and students. Backend for this project stays in [this repo](https://github.com/buyan-m/courses-backend)
+
+The project based on Vue 3, Pinia, Vue CLI and Editorjs.
```
Comment (RuslanKhorolets) [inline README.md:4]:
```suggestion
The project is based on Vue 3, Pinia, Vue CLI, and Editorjs. 
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+## [Okul.one](https://okul.one)
+Free educational platform for teachers and students. Backend for this project stays in [this repo](https://github.com/buyan-m/courses-backend)
```
Comment (RuslanKhorolets) [inline README.md:2]:
```suggestion
Free educational platform for teachers and students. The backend for this project stays in [this repo](https://github.com/buyan-m/courses-backend)
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+To get started with the project, follow these steps
+- Run `npm i` to install the necessary dependencies.
+- If you don't want to set up your own backend, you can modify the devServer configuration in the vue.config.js file. If you only want to take a look at the project, you can use the production API.
+- You can also remove HTTPS part to avoid creating a certificate.
+- Run `npm run dev` to start the development server.
+
+If you want to have the full experience, you can use Docker Compose.
+
+#### Technical details
+The project has two major parts: Editor module for creating courses and Viewer module for taking courses.
```
Comment (RuslanKhorolets) [inline README.md:15]:
```suggestion
The project has two major parts: The editor module for creating courses and the Viewer module for taking courses.
```

**Review by RuslanKhorolets — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+## [Okul.one](https://okul.one)
+Free educational platform for teachers and students. The backend for this project stays in [this repo](https://github.com/buyan-m/courses-backend)
+
+The project is based on Vue 3, Pinia, Vue CLI, and Editorjs. 
+#### How to start
+To get started with the project, follow these steps
+- Run `npm i` to install the necessary dependencies.
```
Comment (serzz1990) [inline README.md:7]: Use `npm ci` for install dependences

**Pull request merged**